### ISSUE TITLE
error retrieving resource lock default/kube-fencing-lock

### ISSUE THREAD

**Issue by jsalatiel** [kvaps/kube-fencing#10]
I am trying to deploy as written on quickstart guide:

I am not sure if I should or not create the namespace fencing manually, anyway, i tried creating and not creating.

After run : kubectl apply -f https://github.com/kvaps/kube-fencing/raw/master/deploy/kube-fencing.yaml I gtt the following in the logs:
```
I1030 23:49:21.927215       1 main.go:25] Kube-Fencing Version: 2.1.0
I1030 23:49:21.927265       1 main.go:26] Go Version: go1.13.15
I1030 23:49:21.927269       1 main.go:27] Go OS/Arch: linux/amd64
I1030 23:49:22.335566       1 main.go:68] Registering Components.
I1030 23:49:22.335658       1 main.go:76] Starting the Cmd.
I1030 23:49:22.335898       1 leaderelection.go:241] attempting to acquire leader lease  fencing/kube-fencing-lock...
E1030 23:49:22.341053       1 leaderelection.go:334] error initially creating leader election record: configmaps is forbidden: User "system:serviceaccount:fencing:fencing-controller" cannot create resource "configmaps" in API group "" in the namespace "fencing"
E1030 23:49:25.797255       1 leaderelection.go:334] error initially creating leader election record: configmaps is forbidden: User "system:serviceaccount:fencing:fencing-controller" cannot create resource "configmaps" in API group "" in the namespace "fencing"
...
```

Am I missing something ?

**Comment by kvaps** [comment]
Probable duplicated https://github.com/kvaps/kube-fencing/issues/8, should be fixed in https://github.com/kvaps/kube-fencing/issues/8

**Comment by jsalatiel** [comment]
I have noticed you updated deploy/helm/kube-fencing/templates/controller-rbac.yaml , but not deploy/kube-fencing.yaml.
Only helm will be updated ?

**Comment by kvaps** [comment]
Ah, you're right. Thank you for pointing out!

Should be fixed in 07f50e9b0d3bc2a3ed88b8e8bc75119f71786bfc
